multimodal/src/autogluon/multimodal/models/mmdet_image.py

```diff
@@ -166,6 +178,9 @@ def forward(
         -------
             A dictionary with bounding boxes.
         """
+        # TODO: refactor this to work like forward() in MMDet
+        logger.warning("MMDetAutoModelForObjectDetection.forward() is deprecated since it does not support multi gpu.")
```

Comment:
I guess `realtime_predict` still uses this forward. Consider support realtime_predict later.

Reply:
Sure, added in the TODO here.